packages/sdk/src/client.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { TypeEqual, expectType } from 'ts-expect';
+
+import { RpcParams, endpoints, getAddresses } from '@leather.io/rpc';
+
+import { createLeatherClient } from './client';
+
+vi.stubGlobal('LeatherProvider', {
+  request: () => Promise.resolve(),
+});
+
+describe('Leather SDK', () => {
+  const client = createLeatherClient();
+
+  const clientMethods = Object.keys(client);
+  test('that it has a method for each endpoint', () => {
+    Object.values(endpoints).forEach(endpoint => clientMethods.includes(endpoint.method));
+  });
+
+  test('Types', async () => {
+    // Optional params, compiler should be happy no parameters
```

Comment:
Maybe you could move these comments into `test('` declarations instead? Then you'd get more feedback on what failed

e.g. 
```
test('Types: compiler should be happy no parameters', async () => {
...

test('Types: Sign message should cause a type error', async () => {
```